src/api/constants/index.ts

```diff
@@ -14,6 +14,20 @@ export const ANSWERS = {
 
 export const GBP_CURRENCY_CODE = 'GBP';
 
+/**
+ * DATE_5_MINUTES_FROM_NOW
+ * Generate a date that is 5 minutes from now
+ * @returns {Date}
+ */
+const DATE_5_MINUTES_FROM_NOW = () => {
```

Comment:
should constants have functions? or should it be in a helper with referral to the milliseconds constant?

Reply:
I don't see why constants shouldn't have functions? We have functions for lots of account things - session expiry, verification expiry, password reset